sql/database.sql

```diff
@@ -5376,6 +5376,7 @@ CREATE TABLE `patient_data` (
   `financial_review` datetime default NULL,
   `pubpid` varchar(255) NOT NULL default '',
   `pid` bigint(20) NOT NULL default '0',
+  `uuid` char(36) NOT NULL default '',
```

Comment:
Per RFC 4122, uuid is 16 octets.  MySQL team has a good example in [this post](https://mysqlserverteam.com/mysql-8-0-uuid-support/).  8.0 mysql example is

> b BINARY(16)  DEFAULT (UUID_TO_BIN(UUID())),